extensions/arc/package.json

```diff
@@ -738,26 +738,66 @@
                           "min": 1,
                           "max": 65535
                         },
+                        {
+                          "label": "%arc.postgres.server.group.engine.version%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_ENGINE_VERSION",
+                          "type": "options",
+                          "options" : ["11","12"],
+                          "defaultValue": "12"
+                        },
+                        {
+                          "label": "%arc.postgres.server.group.extensions.label%",
+                          "description": "%arc.postgres.server.group.extensions.description%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_EXTENSIONS",
+                          "type": "text"
+                        }
+
+                      ]
+                    },
+                    {
+                      "title": "%arc.postgres.settings.storage.title%",
+                      "fields": [
                         {
                           "label": "%arc.storage-class.data.label%",
                           "description": "%arc.postgres.storage-class.data.description%",
                           "variableName": "AZDATA_NB_VAR_POSTGRES_STORAGE_CLASS_DATA",
                           "type": "kube_storage_class",
                           "required": true
                         },
+                        {
+                          "label": "%arc.postgres.server.group.volume.size.data.label%",
+                          "description": "%arc.postgres.server.group.volume.size.data.description%",
+                          "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_VOLUME_SIZE_DATA",
+                          "type": "text",
+                          "defaultValue": "5Gi"
```

Comment:
Question for @yualan  or maybe @chlafreniere on what the unit should be for the volume size.  The CLI accepts a [Kubernetes resource quantity](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#resource-quantities) (an integer followed by a storage unit like Ki, Mi, Gi, Ti).  We could accept that format and write a regex to validate it.

But I noticed that we handled RAM differently.  Instead of making them input '1024Mi', the field documents that the unit is megabytes and accepts the integer '1024'.  Then we append 'Mi' when calling the CLI.  Should we do the same here with gigabytes as the unit?

Reply:
To keep it simple, let's use the same for gigabytes. Confirmed with @twright-msft